### DISCUSSION TITLE
Volume rendering tab not visible

### DISCUSSION THREAD

**Discussion by carlachristolin** [mlavik1/UnityVolumeRendering#157] [Q&A]
Hello,
I just downloaded your code from github and into Unity. However, when I open up the project, I do not see the Volume Rendering tab in the menu bar as described in your tutorial. Furthermore, I also get an error in the console saying "Assets\3rdparty\Nifti.NET\NiftiFile.cs(489,53): error CS1069: The type name 'Color' could not be found in the namespace 'System.Drawing'. This type has been forwarded to assembly 'System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' Consider adding a reference to that assembly."
I was wondering if you could help me with this issue. 
Thank you

**Comment by mlavik1** [comment]
Hi!, and thanks for reporting!
I haven't seen that issue before. I have a few questions:
- Which version of Unity are you using?
- Which platform are you building for? (the "target" in [build settings](https://docs.unity3d.com/Manual/BuildSettings.html))
- Are you suing .NET 4.X? (check here: https://learn.microsoft.com/en-us/visualstudio/gamedev/unity/unity-scripting-upgrade) or 2.0, or IL2CPP?

You could try out what they describe in this article, to see if that fixes the issue for you: https://docs.unity3d.com/Manual/dotnetProfileAssemblies.html (but with "System.Drawing.Primitives.dll" instead of the DLL they use in the example)
Basically, just create a file called csc.rsp in the Assets folder, and add this line to it: 
```
-r:System.Drawing.Primitives.dll
```

Another thing that might work is to switch from .NET 4.x to 2.0 (as described here: https://docs.unity3d.com/Manual/BuildSettings.html), though that might not be an ideal solution for you.

I'll see if I can reproduce it too, and implement a proper fix for the issue.
